service/share/org.opensuse.DInstaller.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Name=org.opensuse.DInstaller
+Exec=/usr/bin/d-installer manager
+User=root
+SystemdService=d-installer.service
```

Comment:
Ha! Adding this line broke the startup:


> Nov 10 05:14:01 dhcp116.suse.cz dbus-daemon[701]: [system] Activating via systemd: service name='org.opensuse.DInstaller' unit='d-installer.service' requested by ':1.101' (uid=0 pid=3514 comm="cockpit-bridge")
Nov 10 05:14:01 dhcp116.suse.cz dbus-daemon[701]: [system] Activation via systemd failed for unit 'd-installer.service': Unit d-installer.service not found.

Reply:
(solved by installing also the systemd service in `build.sh`)